src/server/CodeStyle/CodeStyle.py

```
@@ -1,4 +1,5 @@
 from antlr4 import *
+from torch.distributions import continuous_bernoulli
```

Ruff (0.11.9)

`torch.distributions.continuous_bernoulli` imported but unused

Remove unused import: `torch.distributions.continuous_bernoulli`

(F401)